### PR TITLE
[ID] Update News styling criteria

### PR DESCRIPTION
**Pull request by reinosk**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by reinosk — COMMENTED** [review]
i fix spaces in the link.

**Review by reinosk — COMMENTED** [review]
I've reviewed it again.

**Review by reinosk — COMMENTED** [review]
i've reviewed again

File: wiki/News_styling_criteria/id.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+- `date` harus mewakili string gabungan yang menampilkan tanggal kalender dalam format ISO 8601 (`2017-08-17`), diikuti dengan string waktu 24 jam (`03:00:00`), diikuti dengan selisih waktu dari UTC (`+0000`). Ini adalah tanggal publikasi yang digunakan untuk menentukan kapan postingan berita dapat dilihat di situs web.
+- Paragraf pratinjau adalah apa yang akan ditampilkan di halaman depan, arsip berita, dan daftar berita dalam game osu!(lazer), selain menjadi paragraf pertama dari postingan berita.
+
+### Markdown
+
+Penggunaan markdown tercakup dalam [kriteria gaya artikel](/wiki/Article_styling_criteria), namun poin berikut ini khususnya berkaitan dengan artikel berita:
+
+- Judul tingkat 1 (`#`), yang berkaitan dengan suatu judul, tidak boleh digunakan, karena judul terdapat pada bagian depan di bagian atas artikel.
+- [Kotak info](/wiki/Article_styling_criteria/Formatting#infoboxes) dan [catatan kaki](/wiki/Article_styling_criteria/Formatting#footnotes) tidak didukung.
+- Gambar [spanduk](#spanduk) tidak boleh berisi teks alternatif apa pun, yaitu teks di dalam tanda kurung siku pada tautan gambar di markdown (`![alt text](image.png)`).
```
Comment (reinosk) [inline wiki/News_styling_criteria/id.md:82]: i fix spaces in the link.

**Review by reinosk — COMMENTED** [review]
everything looks ok

**Comment by iSlodinxOsu** [comment]
Please change the title to `[ID] Update News styling criteria`

**Comment by reinosk** [comment]
> Please change the title to `[ID] Update News styling criteria`

Okay done

**Review by TicClick — APPROVED** [review]
merging (no native review for one week or longer) while ignoring the above notice (an update of the original page) since it may put this PR on hold for even longer

**Pull request merged**